### PR TITLE
Formatting and docstring

### PR DESCRIPTION
**Pull request by lakshyacis**
Formatting and docstring.
Flake8 and pylint fixes.

### CODE REVIEW

File: src/class_smtpDeliver.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 """
-src/class_smtpDeliver.py
-========================
+Deliver threads
```
Comment (PeterSurda) [inline src/class_smtpDeliver.py]: SMTP client thread for delivering emails

File: src/multiqueue.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 """
-src/multiqueue.py
-=================
+Manage multiple queues
```
Comment (PeterSurda) [inline src/multiqueue.py]: A queue with multiple internal subqueues. Elements are added into a random subqueue, and retrieval rotates

File: src/proofofwork.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # pylint: disable=too-many-branches,too-many-statements,protected-access
 """
-src/proofofwork.py
-==================
+Get proof after work done
```
Comment (PeterSurda) [inline src/proofofwork.py]: Proof of work calculation

File: src/multiqueue.py (hunk 2 above)
Comment (lakshyacis) [reply src/multiqueue.py]: Okay. Thank you

**Pull request merged**